geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateGatewaySenderCommand.java

```diff
@@ -51,25 +51,32 @@
       interceptor = "org.apache.geode.management.internal.cli.commands.CreateGatewaySenderCommand$Interceptor")
   @ResourceOperation(resource = ResourcePermission.Resource.CLUSTER,
       operation = ResourcePermission.Operation.MANAGE, target = ResourcePermission.Target.GATEWAY)
-  public ResultModel createGatewaySender(@CliOption(key = {CliStrings.GROUP, CliStrings.GROUPS},
-      optionContext = ConverterHint.MEMBERGROUP,
-      help = CliStrings.CREATE_GATEWAYSENDER__GROUP__HELP) String[] onGroups,
+  public ResultModel createGatewaySender(
+
+      @CliOption(key = {CliStrings.GROUP, CliStrings.GROUPS},
+          optionContext = ConverterHint.MEMBERGROUP,
+          help = CliStrings.CREATE_GATEWAYSENDER__GROUP__HELP) String[] onGroups,
 
       @CliOption(key = {CliStrings.MEMBER, CliStrings.MEMBERS},
           optionContext = ConverterHint.MEMBERIDNAME,
           help = CliStrings.CREATE_GATEWAYSENDER__MEMBER__HELP) String[] onMember,
 
-      @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__ID, mandatory = true,
+      @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__ID,
+          mandatory = true,
           help = CliStrings.CREATE_GATEWAYSENDER__ID__HELP) String id,
 
-      @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__REMOTEDISTRIBUTEDSYSTEMID, mandatory = true,
+      @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__REMOTEDISTRIBUTEDSYSTEMID,
+          mandatory = true,
           help = CliStrings.CREATE_GATEWAYSENDER__REMOTEDISTRIBUTEDSYSTEMID__HELP) Integer remoteDistributedSystemId,
 
-      @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__PARALLEL, specifiedDefaultValue = "true",
+      @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__PARALLEL,
+          specifiedDefaultValue = "true",
           unspecifiedDefaultValue = "false",
           help = CliStrings.CREATE_GATEWAYSENDER__PARALLEL__HELP) boolean parallel,
 
       @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__MANUALSTART,
+          specifiedDefaultValue = "false",
```

Comment:
I think this should be `"true"` - no?

Reply:
I actually had doubts about this one as it "breaks" the purpose of the PR... however, this flag is actually [`deprecated`](https://geode.apache.org/docs/guide/18/tools_modules/gfsh/command-pages/create.html#topic_hg2_bjz_ck) and users must avoid using it at all cost: it might cause data loss and several other issues during startup, that's why I just left it as `false` on purpose. Long story short, if a user wants to enable `manual-start`, they actually need to know exactly what they're doing and specify the entire argument as `--manual-start=true`.
Just my two cents, I can revert it if you don't agree with the above.

Reply:
That's great info! Please add it as a comment to the option so that the next person looking at the code doesn't need to wonder why :)

Reply:
👍

Reply:
if we want user always specify a value when they specify this option. Let's not provide a default value to it.

Reply:
Fair enough, I will `force push` the change shortly.